NAMESPACE

```diff
@@ -6,6 +6,7 @@ export(icd)
 export(iget)
 export(ils)
 export(imeta)
+export(imkdir)
```

Comment:
this commit can reference issue 1 with a # sign when you're ready - and then the work will be marked back to open issues.

Reply:
Done